starwars_api/models.py

```diff
@@ -28,8 +33,11 @@ def all(cls):
         later in charge of performing requests to SWAPI for each of the
         pages while looping.
         """
-        pass
-
+        if cls.RESOURCE_NAME == 'people':
+            return PeopleQuerySet()
+        if cls.RESOURCE_NAME == 'films':
+            return FilmsQuerySet()
+        
 
 class People(BaseModel):
```

Comment:
To handle the unicode characters in some names you can add this to your __repr__

    self.name = self.name.encode('utf8', 'ignore')